pages/team.jsx

```diff
@@ -1,76 +1,383 @@
+  },
+  {
+    name: `Irini Papadimitriou`,
+    twitter: ``,
+    // pic: `/assets/images/team/wrangler/`,
+    bio:  (<div>
+            <p>Irini Papadimitriou is Digital Programmes Manager at the V&A and responsible for programmes such as the annual Digital Design Weekend, a big-scale event presenting intersections of art, design and technology; and the monthly Digital Futures, an open platform & networking event for displaying and discussing work by researchers, artists, designers, companies and other professionals working with art, technology, design, science and beyond.</p>
+            <p>Irini is also Head of New Media Arts Development at Watermans, an arts organisation presenting innovative work and supporting artists working with technology, where she is curating the exhibition programme and an annual Digital Performance festival, exploring intersections in digital art, dance, sound, performance, science and more.</p>
+            <p>She is also one of the organisers for London's Mini Maker Faire, and one of the co-founders of Maker Assembly, a critical gathering about maker cultures.</p>
+            <p>Links
+              <ul>
+                <li><a href="http://www.vam.ac.uk/blog/author/irini-papadimitriou">V&A blog</a></li>
+                <li><a href="https://www.watermans.org.uk/new-media-arts-archive/">Watermans New Media Arts</a></li>
+                <li><a href="http://makerassembly.org ">Maker Assembly</a></li>
+              </ul>
+            </p>
+          </div>)
+  }
 ];
 
-var MemberProfile = React.createClass({
+var Partners = React.createClass({
+  partnersInfo: [
+    {
+      name: `EPIK`,
+      logo: `/assets/images/team/partner/EPIK.jpg`,
+      link: `http://epik.org.uk/`
+    },
+    {
+      name: `The Tate`,
+      logo: `/assets/images/team/partner/Tate.jpg`,
+      link: `http://www.tate.org.uk/`
+    },
+    {
+      name: `The V&A`,
+      logo: `/assets/images/team/partner/VA.jpg`,
+      link: `https://www.vam.ac.uk/`
+    },
+    {
+      name: `Arts Award`,
+      logo: `/assets/images/team/partner/ArtsAward.jpg`,
+      link: `http://www.artsaward.org.uk/`
+    },
+    {
+      name: `BBC`,
+      logo: `/assets/images/team/partner/BBC.jpg`,
+      link: `http://www.bbc.co.uk/rd`
+    },
+    {
+      name: `Ravensbourne`,
+      logo: `/assets/images/team/partner/Ravensbourne.jpg`,
+      link: `http://www.ravensbourne.ac.uk/`
+    },
+    {
+      name: `Digital Me`,
+      logo: `/assets/images/team/partner/DigitalMe.jpg`,
+      link: `http://www.digitalme.co.uk/`
+    }
+  ],
   render: function() {
+    let groups = [];
+    let tempGroup = [];
+
+    this.partnersInfo.forEach((partner) => {
+      let thePartner = ( <a href={partner.link}>
```

Comment:
I don't generally use `target="_blank"` as opening links in new window can be disorienting to people. Accessibility wise it's also generally good to avoid using it. 😁